### PR TITLE
data trails: alternate legend display for missing label value

### PR DESCRIPTION
**Pull request by darrenjaneczek**
**What is this feature?**

When the data doesn't have a label value, the default "Value" is used.
image

This PR replaces it with `<unspecified ${label}>`
image

This is similar to what we do in the title of the label-specific breakdown view:

image

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by catherineymgui — APPROVED** [review]

File: public/app/features/trails/ActionTabs/BreakdownScene.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
   VariableDependencyConfig,
   VizPanel,
 } from '@grafana/scenes';
+import { DataQuery } from '@grafana/schema/dist/esm/index';
```
Comment (NWRichmond) [inline public/app/features/trails/ActionTabs/BreakdownScene.tsx:24]: We should be fine to import directly from `'@grafana/schema'` here:

```suggestion
import { DataQuery } from '@grafana/schema';
```

File: public/app/features/trails/ActionTabs/BreakdownScene.tsx [hunk 2]
```diff
@@ -456,3 +458,27 @@ function getBreakdownSceneFor(model: SceneObject): BreakdownScene {
...
+
+function fixLegendForUnspecifiedLabelValueBehavior(vizPanel: VizPanel) {
+  vizPanel.state.$data?.subscribeToState((newState, prevState) => {
+    const target = newState.data?.request?.targets[0];
+    if (hasLegendFormat(target)) {
+      const { legendFormat } = target;
+      // Assume {{label}}
+      const label = legendFormat.slice(2, -2);
+
+      newState.data?.series.forEach((series, index) => {
```
Comment (NWRichmond) [inline public/app/features/trails/ActionTabs/BreakdownScene.tsx:470]: `index` is unused:

```suggestion
      newState.data?.series.forEach((series) => {
```

**Review by NWRichmond — COMMENTED** [review]
Nice! Just a couple of small suggestions:

**Review by NWRichmond — APPROVED** [review]
Nicely done!

**Pull request merged**